lib/matplotlib/backends/_backend_pdf_ps.py

```diff
@@ -2,20 +2,58 @@
 Common functionality between the PDF and PS backends.
 """
 
+from io import BytesIO
 import functools
+import logging
+
+from fontTools import log as ftlog
+from fontTools import subset
 
 import matplotlib as mpl
 from .. import font_manager, ft2font
 from ..afm import AFM
 from ..backend_bases import RendererBase
 
 
+# only log critical errors from fontTools
+ftlog.setLevel(logging.CRITICAL)
```

Comment:
Someone could see it as bad behaviour from a library if it modifies the logging configuration for another library. If this is about the unknown truetype table, I think it's fine to have our own list of dropped tables and add to it whenever someone complains about messages with a new font.

Reply:
That makes sense.
I'll fix this in the next commit